layouts/partials/social.html

```diff
@@ -38,6 +38,12 @@
       <a class="pure-menu-link" href="http://weibo.com/{{ . }}" target="_blank"><i class="fa fa-weibo fa-fw"></i>Weibo</a>
     </li>
     {{ end }}
+	
+	{{ with .Site.Social.tumblr }}
```

Comment:
Thanks @Confirm4Crit 

Please could you change this indentation so that it matches with the closing `{{ end }}`?